cirq-superstaq/cirq_superstaq/job_test.py

```diff
@@ -134,6 +136,22 @@ def test_compiled_circuit(job: css.job.Job) -> None:
     assert job.compiled_circuits(index=0) == compiled_circuit
 
 
+def test_pulse_gate_circuit(job: css.job.Job) -> None:
+    pulse_gate_circuit = qiskit.QuantumCircuit(1, 1)
+    pulse_gate = qiskit.circuit.Gate("test_pulse_gate", 1, [3.14, 1])
+    pulse_gate_circuit.append(pulse_gate, [0])
+    pulse_gate_circuit.measure(0, 0)
+    job_dict = {"status": "Done", "pulse_gate_circuits": qss.serialize_circuits(pulse_gate_circuit)}
```

Comment:
we already have a function in compiler_output.py (`_deserialize_qiskit_circuits`) to deserialize qiskit circuits, with special handling for qiskit not being installed/etc. - let's move to `css.serialization.deserialize_qiskit_circuits` so we can use it here too